tests/unit/test_ops.py

```diff
@@ -406,3 +406,119 @@ def test_dropna(tmpdir, datasets, engine="parquet"):
         new_gdf = dropna.apply_op(gdf, columns_ctx, "all")
         assert new_gdf.columns.all() == gdf.columns.all()
         assert new_gdf.isnull().all().sum() < 1, "null values exist"
+
+
+@pytest.mark.parametrize("gpu_memory_frac", [0.01, 0.1])
+@pytest.mark.parametrize("engine", ["parquet", "csv", "csv-no-header"])
+@pytest.mark.parametrize("op_columns", [["x"], None])
+def test_normalize(tmpdir, datasets, gpu_memory_frac, engine, op_columns):
+    paths = glob.glob(str(datasets[engine]) + "/*." + engine.split("-")[0])
+
+    if engine == "parquet":
+        df1 = cudf.read_parquet(paths[0])[mycols_pq]
+        df2 = cudf.read_parquet(paths[1])[mycols_pq]
+    else:
+        df1 = cudf.read_csv(paths[0], header=False, names=allcols_csv)[mycols_csv]
+        df2 = cudf.read_csv(paths[1], header=False, names=allcols_csv)[mycols_csv]
+    df = cudf.concat([df1, df2], axis=0)
+    df["id"] = df["id"].astype("int64")
+
+    if engine == "parquet":
+        cat_names = ["name-cat", "name-string"]
+        columns = mycols_pq
+    else:
+        cat_names = ["name-string"]
+        columns = mycols_csv
+    cont_names = ["x", "y", "id"]
+    label_name = ["label"]
+
+    data_itr = nvtabular.io.GPUDatasetIterator(
+        paths,
+        columns=columns,
+        use_row_groups=True,
+        gpu_memory_frac=gpu_memory_frac,
+        names=allcols_csv,
+    )
+
+    config = nvt.workflow.get_new_config()
+    config["PP"]["continuous"] = [ops.Moments()]
+
+    processor = nvt.Workflow(
+        cat_names=cat_names,
+        cont_names=cont_names,
+        label_name=label_name,
+        config=config,
+        to_cpu=False,
+    )
+
+    processor.update_stats(data_itr)
```

Comment:
Can you update this to be against the latest master? I refactored the workflow/unittests last week . The workflow object no longer has a 'to_cpu' flag (#94) and the unittests have fixtures for creating the datasets / test dataframe (#106) to avoid some of the boilerplate in creating tests:

With these changes this should be something like:

```suggestion
def test_normalize(tmpdir, df, dataset, gpu_memory_frac, engine, op_columns):
    cat_names = ["name-cat", "name-string"] if engine == "parquet" else ["name-string"]
    cont_names = ["x", "y", "id"]
    label_name = ["label"]

    config = nvt.workflow.get_new_config()
    config["PP"]["continuous"] = [ops.Moments()]

    processor = nvt.Workflow(
        cat_names=cat_names,
        cont_names=cont_names,
        label_name=label_name,
        config=config,
    )

    processor.update_stats(dataset)
```
(and likewise for test_normalize_minmax)